frontend/src/routes/_dashboard/index.lazy.tsx

```diff
@@ -19,8 +60,181 @@ function AuthedView() {
+            className="px-14 placeholder:text-gray-400  -mx-5 w-full my-1 text-sm focus-visible:right-0"
+            placeholder="Ex: ZaneOps"
+          />
+        </div>
+
+        <div className="md:w-fit w-full">
+          <Menubar className="border border-border md:w-fit w-full ">
+            <MenubarMenu>
+              <MenubarTrigger className="flex md:w-fit w-full ring-secondary md:justify-center justify-between text-sm items-center gap-1">
+                Status
+                <ChevronsUpDown className="w-4" />
+              </MenubarTrigger>
+              <MenubarContent className=" border min-w-[23rem] border-border md:min-w-6">
+                <MenubarContentItem icon={Rocket} text="Active" />
+                <MenubarContentItem icon={Trash} text="Archived" />
+              </MenubarContent>
+            </MenubarMenu>
+          </Menubar>
+        </div>
+      </div>
+
+      <Table>
+        <TableHeader className="bg-toggle">
+          <TableRow className="border-none">
+            <TableHead>Name</TableHead>
+            <TableHead className="flex items-center gap-2">
+              Last Updated
+              <ArrowDown size={15} />
+            </TableHead>
+            <TableHead>Status</TableHead>
+            <TableHead className="text-right">Actions</TableHead>
+          </TableRow>
+        </TableHeader>
+        <TableBody>
+          {projects.map((project) => (
+            <TableRow
+              className="border-border cursor-pointer"
+              key={project.name}
+            >
+              <TableCell className="font-medium flex items-center gap-3">
+                <Folder size={18} />
+                {project.name}
+              </TableCell>
+              <TableCell>{project.updated_at}</TableCell>
+              <TableCell>
+                <div
+                  className={`flex border md:w-fit w-40 px-3 py-1 border-opacity-60 rounded-full text-sm items-center gap-2 ${
+                    project.tracker === 1
+                      ? "border-green-600  bg-green-600 bg-opacity-10 text-statusgreen"
+                      : project.tracker === 0
+                        ? "border-red-600 bg-red-600 bg-opacity-10 text-statusred"
+                        : project.tracker === 2
+                          ? "border-yellow-600 bg-yellow-600 bg-opacity-10 text-statusyellow"
+                          : ""
+                  }`}
+                >
+                  <div
+                    className={`border w-2 h-2 text-white  border-transparent p-0.5 rounded-full ${
+                      project.tracker === 1
+                        ? "bg-green-600"
+                        : project.tracker === 0
+                          ? "bg-red-600"
+                          : project.tracker === 2
+                            ? "bg-yellow-600"
+                            : ""
+                    }`}
+                  ></div>
+                  {project.status}
+                </div>
```

Comment:
This should be in a component, named `chip` or `badge`. 

It can accept the a `color` property which can be `red`, `green` and `yellow` and accept `children` for the text inside.